src/main/java/homework6/entity/JsonData.java

```diff
@@ -0,0 +1,57 @@
+package homework6.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JsonData {
+
+    @JsonProperty("data_1")
```

Comment:
This is not the great solution to create the class with reference to data count in the file. Just imagine the situation that amount of data is changed ?